src/v1/camera.rs

```diff
@@ -24,7 +37,7 @@ pub struct Perspective {
     /// The floating-point distance to the near clipping plane.
     ///
     /// `zfar` must be greater than `znear`.
-    pub znear: f32, 
+    pub znear: f32,
```

Comment:
I prefer `z_near` etc but that's really just a matter of style. I think 'proper' snake_case actually mandates no single letter words at the start (hence why standard library has `btree_map` and not `b_tree_map`) but in this particular case `z_near` just seems so much easier on the eye than `znear`.